clients/admin-ui/src/features/integrations/configure-monitor/MonitorConfigActionsCell.tsx

```diff
@@ -0,0 +1,20 @@
+import { EditIcon, HStack, IconButton } from "fidesui";
+
+const MonitorConfigActionsCell = ({
+  onEditClick,
+}: {
+  onEditClick: () => void;
+}) => (
+  <HStack onClick={(e) => e.stopPropagation()} cursor="auto">
```

Comment:
I recently added a new `meta` property to our tables to help prevent the need for this kind of thing. If you want to merge in latest from `main` and update the columnHelper for this cell to include:
```meta: { disableRowClick: true }```